src/PiggyBank/PiggyBank.Lambda/Request.cs

```diff
@@ -9,14 +11,22 @@ public Request(APIGatewayProxyRequest request)
         {
             Method = new HttpMethod(request.HttpMethod);
             Path = request.Path;
+            Body = request.Body;
         }
 
         public HttpMethod Method { get; }
         public string Path { get; }
+        public string Body { get; }
 
         public string GetEndpoint() 
         {
             return Method + Path;
         }
+
+        public T DeserializeBody<T>()
+        {
+            Console.WriteLine("#######" + Body);
```

Comment:
Oops we forgot to remove this - my bad, should have done when you said 😁

Reply:
🤣